index.bs

```diff
@@ -405,38 +400,66 @@ spec: webidl
     };
   </pre>
   <p>
-    The steps to <dfn>retrieve a permission storage entry</dfn> of a
-    <a>permission storage identifier</a> are as follows:
+    The steps to <dfn export>retrieve the permission storage</dfn> of a
+    {{PermissionName}} <var>name</var> are as follows:
   </p>
   <ol>
-    <li>If the <a>user agent</a> has a {{PermissionStorage}} associated
-    with the <a>permission storage identifier</a> in its permission store,
-    it MUST return the {{PermissionStorage}}.
+    <li>Let <var>settings</var> be the <a>current settings object</a>.</li>
+    <li>
+      Let <var>store</var> be some <a>permission store</a> associated with the
+      <a>same origin</a> as <var>settings</var>. This store SHOULD be chosen
+      consistently for a given <a>environment settings object</a>.
```

Comment:
Yeah, I think that works. I shied away from it because @annevk didn't want to have per-realm stores at all, but once we allow them, requiring them in the spec doesn't actually require implementations to store them separately.